packages/language-server/src/completion-items.ts

```diff
@@ -190,13 +256,63 @@ function getClassNamespacePrefix(
   return undefined;
 }
 
+function getNodeFilterText(suggestion: UI5XMLViewCompletion): string {
```

Comment:
What happens if we have no filterText at all? is there a default value?

Reply:
Yes, the label is used (and then suggestions that don't contain the entire text that is being replaced, like FQN or namespace, will be filtered out)